packages/gatsby-transformer-remark/src/__tests__/__snapshots__/extend-node.js.snap

```diff
@@ -631,7 +631,9 @@ Object {
 <li><a href=\\"/my%20little%20pony/#second-title\\">second title</a></li>
 </ul>
 </li>
-<li><a href=\\"/my%20little%20pony/#third-title\\">third title</a></li>
+<li>
+<p><a href=\\"/my%20little%20pony/#third-title\\">third title</a></p>
+</li>
```

Comment:
This is a pre-13.0 change (happens even with 12.0) Makes sense to me because now top-level list items have consistent markup (expand above to compare with the item having a sub-list)